addons/hr_timesheet/models/project.py

```diff
@@ -327,6 +327,32 @@ def _compute_subtask_effective_hours(self):
         for task in self.with_context(active_test=False):
             task.subtask_effective_hours = sum(child_task.effective_hours + child_task.subtask_effective_hours for child_task in task.child_ids)
 
+    def get_group_pattern(self):
+        group_pattern = super().get_group_pattern()
+        group_pattern['planned_hours'] = r'\s(\d+(?:\.\d+)?)[hH]'
+        return group_pattern
```

Comment:
Isn't it equivalent to :
```suggestion
        return {
            **super().get_group_pattern(),
            'planned_hours': r'\s(\d+(?:\.\d+)?)[hH]',
        }
```

Reply:
yes, thank your all your comments, they are useful for a new pythonist with a c++ coding mind/style